data/maps/Dolomites_v3.wmf/elemental

```diff
@@ -8,5 +8,25 @@ nr_players="8"
 name=_"Dolomites 3.0"
 author="King of nowhere"
 descr=_"The jewel of the alps, the dolomites mountain range has been declared a world heritage site for its pristine beauty. This map is an as-faithful-as-reasonable reconstruction of the area delimited by Cortina d'Ampezzo in the north-east and Canazei in the south-west. If you have to be pitted in a duel to the death with your enemies, it may as well be in a beautiful place."
-hint=_"- there is lot of flat ground on the mountains. Unfortunately, most of it is vertical. Use wisely the little building space you have. - A fortress at the mouth of a narrow mountain pass can be a formidable obstacle to an enemy army. Use the terrain to your advantage - chop all trees and expand wherever you can; it's the only way to find all the precious farm spaces - suggested teams: red+orange+blue+violet"
+hint=_"There is lot of flat ground on the mountains. Unfortunately, most of it is vertical - use wisely the little building space you have. A fortress at the mouth of a narrow mountain pass can be a formidable obstacle to an enemy army - use the terrain to your advantage. Chop all trees and expand wherever you can - it's the only way to find all the precious farm spaces."
```

Comment:
Please use `–` instead of `-`. Otherwise LGTM

Reply:
Well, what is the difference and how to type this sign?

Reply:
The difference is consistency with all other strings. On Linux this is typed with `AltGr`+`-`; Windows will probably allow `Alt`+`2013` (alternatively copy-paste: – )

Reply:
ok tried to copy but it doesn't show any difference in Notepad++. So if not sufficient I can't help with this

Reply:
That's the right one now :)